src/third_party/cppcodec/CMakeLists.txt

```diff
@@ -9,26 +9,30 @@ include(CTest)
 # In your own project that uses cppcodec, you might want to specify a different standard or error level.
 
 # Request C++11, or let the user specify the standard on via -D command line option.
-if (NOT CMAKE_CXX_STANDARD)
-  set(CMAKE_CXX_STANDARD 11)
-endif()
-set(CMAKE_CXX_STANDARD_REQUIRED ON)
-set(CMAKE_CXX_EXTENSIONS OFF)
+#if (NOT CMAKE_CXX_STANDARD)
```

Comment:
should most of this be removed if it is not needed?

Reply:
This file is mostly copy/pasted (vendored) from cppcodec, and I figured leaving the lines commented-out would help any poor souls trying to update cppcodec in the future identify what things were changed.